test/expected.txt

```diff
@@ -798,14 +798,14 @@ FAIL	/scroll-animations/scroll-timelines/scroll-animation.html	Sending animation
 FAIL	/scroll-animations/scroll-timelines/scroll-timeline-invalidation.html	Animation current time and effect local time are updated after scroller content size changes.
 PASS	/scroll-animations/scroll-timelines/scroll-timeline-invalidation.html	Animation current time and effect local time are updated after scroller size changes.
 FAIL	/scroll-animations/scroll-timelines/scroll-timeline-invalidation.html	If scroll animation resizes its scroll timeline scroller, layout reruns once per frame.
-FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with percentage range [JavaScript API]
+PASS	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with percentage range [JavaScript API]
 FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with px range [JavaScript API]
 FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with calculated range [JavaScript API]
 FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with EM range [JavaScript API]
-FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with percentage range [CSS]
-FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with px range [CSS]
-FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with calculated range [CSS]
-FAIL	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with EM range [CSS]
+TIMEOUT	/scroll-animations/scroll-timelines/scroll-timeline-range.html	Scroll timeline with percentage range [CSS]
```

Comment:
I tried to look into why these tests time out. It seems like a combination of lenient parsing, and reuse of the source elements in the tests. 

We parse and store values for range start/end that we cannot resolve yet, such as `5em`.
We then fail to auto-align the start time, as the offsets can't be resolved, and throw an error.

We could probably both
- be stricter in values we accept and throw a TypeError straight away when setting the value
- handle runtime errors in stead leaving the animation in an unexpected state


This is not something we need to handle now, but I can open a separate PR for handling unresolvable ranges when auto-aligning the start time.